### PR TITLE
Update from auto-update/renku-ui-2.15.0

### PR DESCRIPTION
**Pull request by RenkuBot**
This is an automated pull request.

/deploy #persist #cypress

### CODE REVIEW

**Review by aledegano — APPROVED** [review]

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,5 +1,42 @@
 .. _changelog:
 
+
+0.22.0
+------
+
+Renku ``0.22.0`` improves the Renku experience when working on RenkuLab.
```
Comment (lokijuhy) [inline CHANGELOG.rst:7]:
```suggestion
Renku ``0.22.0`` brings back project and dataset images.
```

File: CHANGELOG.rst [hunk 2]
```diff
@@ -1,5 +1,42 @@
...
+
+User-Facing Changes
+~~~~~~~~~~~~~~~~~~~
+
+**✨ Improvements**
+
+* 🔲 **UI**: Improve styling of buttons with menu and group buttons
+  (`#2243 <https://github.com/SwissDataScienceCenter/renku-ui/pull/2243>`_).
+* 👤 **UI**: Limit the available namespaces for new projects to owned-only to prevent failures
+  (`#2187 <https://github.com/SwissDataScienceCenter/renku-ui/pull/2187>`_).
```
Comment (lokijuhy) [inline CHANGELOG.rst:17]: This sounds like a bug to me? Also, are you referring to namespaces available at project creation? If so, I would reword to `Limit the available namespaces in which new projects may be created to one ones owned by the user to prevent failures`

File: CHANGELOG.rst [hunk 3]
```diff
@@ -1,5 +1,42 @@
...
+
+**🐞 Bug Fixes**
+
+* ✒️ **UI**: Restore per-language source highlighting in the file preview
+  (`#2233 <https://github.com/SwissDataScienceCenter/renku-ui/pull/2233>`_,
+  `#2265 <https://github.com/SwissDataScienceCenter/renku-ui/pull/2265>`_).
+* 🖼️ **UI**: Show projects and datasets avatars again
+  (`#2231 <https://github.com/SwissDataScienceCenter/renku-ui/pull/2231>`_).
```
Comment (lokijuhy) [inline CHANGELOG.rst:25]: I think this counts as an improvement, not a bug!

**Review by lokijuhy — CHANGES_REQUESTED** [review]
General formatting comment: I have decided not to include emojis for each item in the Bug Fixes section- it's a little much, even for me 😋 Please update

**Comment by lorenzo-cavazzi** [comment]
@lokijuhy Thank you for the review!

Just for the record, I don't support removing emojis from the bugfix section but I won't oppose it because I ~~want to merge the PR soon~~ trust your judgment.

**Review by lokijuhy — APPROVED** [review]

**Review by aledegano — APPROVED** [review]
